shannon_energy_rpd.py

```diff
@@ -131,7 +131,7 @@ def running_mean(x, N):
     highs = np.arange(i+1, i+search_window_half+1)
     if highs[-1] > len(EKG):
         highs = np.delete(highs, np.arange(np.where(highs == len(EKG))[0], len(highs)+1))
-    ekg_window = np.concatenate(lows, i, highs)
+    ekg_window = np.concatenate([lows, i, highs])
```

Comment:
I think, was meant to be:
`ekg_window = np.concatenate((lows, i, highs), axis=None)`

Reply:
My bad, I haven't actually been running the script in this repo, I've just been copying out the algorithm and running it on my data.

Actually it was meant to be this:
```
ekg_window = np.concatenate((lows, [i], highs))
```

Reply:
Got it thanks! Can you update the commit with the new code so that I can merge the request?